test.c

```diff
@@ -0,0 +1,99 @@
+#include "wslib.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+
+void main(int argc, char *argv[]) {

```

Comment:
```suggestion
int main(int argc, char *argv[]) {

```

Powinno być tak, bo `void` przypadkiem działa - wg standardu stosujemy `int` 
https://stackoverflow.com/a/18928287/384309